src/models/config/PagesAnalyticsConfig.ts

```diff
@@ -37,4 +37,9 @@ export interface PagesAnalyticsConfig extends BaseAnalyticsConfig {
    * The details of the page type
    */
   pageType: DirectoryPage|EntityPage|LocatorPage|StaticPage
+
+  /**
+   * The domain of the page. If none is specified, the hostname for the site ID is used.
```

Comment:
one potential gotcha maybe worth documenting is that for product=sites if you use this param it must include the protocol for the domain. If you're fire ktag events the protocol will be added for you automatically.